Tools/rdmp/CommandLine/Gui/ConsoleGuiSqlEditor.cs

```diff
@@ -313,5 +326,167 @@ protected virtual void OnQueryCompleted(DataTable dt)
+                keywords.Add("top");
+                keywords.Add("from");
+                keywords.Add("create");
+                keywords.Add("CIPHER");
+                keywords.Add("CLASS_ORIGIN");
+                keywords.Add("CLIENT");
+                keywords.Add("CLOSE");
+                keywords.Add("COALESCE");
+                keywords.Add("CODE");
+                keywords.Add("COLUMNS");
+                keywords.Add("COLUMN_FORMAT");
+                keywords.Add("COLUMN_NAME");
+                keywords.Add("COMMENT");
+                keywords.Add("COMMIT");
+                keywords.Add("COMPACT");
+                keywords.Add("COMPLETION");
+                keywords.Add("COMPRESSED");
+                keywords.Add("COMPRESSION");
+                keywords.Add("CONCURRENT");
+                keywords.Add("CONNECT");
+                keywords.Add("CONNECTION");
+                keywords.Add("CONSISTENT");
+                keywords.Add("CONSTRAINT_CATALOG");
+                keywords.Add("CONSTRAINT_SCHEMA");
+                keywords.Add("CONSTRAINT_NAME");
+                keywords.Add("CONTAINS");
+                keywords.Add("CONTEXT");
+                keywords.Add("CONTRIBUTORS");
+                keywords.Add("COPY");
+                keywords.Add("CPU");
+                keywords.Add("CURSOR_NAME");
+                keywords.Add("primary");
+                keywords.Add("key");
+                keywords.Add("insert");
+                keywords.Add("alter");
+                keywords.Add("add");
+                keywords.Add("update");
+                keywords.Add("set");
+                keywords.Add("delete");
+                keywords.Add("truncate");
+                keywords.Add("as");
+                keywords.Add("order");
+                keywords.Add("by");
+                keywords.Add("asc");
+                keywords.Add("desc");
+                keywords.Add("between");
+                keywords.Add("where");
+                keywords.Add("and");
+                keywords.Add("or");
+                keywords.Add("not");
+                keywords.Add("limit");
+                keywords.Add("null");
+                keywords.Add("is");
+                keywords.Add("drop");
+                keywords.Add("database");
+                keywords.Add("table");
+                keywords.Add("having");
+                keywords.Add("in");
+                keywords.Add("join");
+                keywords.Add("on");
+                keywords.Add("union");
+                keywords.Add("exists");
+
+                // HACK to workaround https://github.com/migueldeicaza/gui.cs/pull/1437
+                var prev = Colors.Menu;
+                Colors.Menu = new ColorScheme()
+                {
+                    Normal = Driver.MakeAttribute(Color.Black, Color.Blue),
+                    Focus = Driver.MakeAttribute(Color.Black, Color.Cyan),
+                };
+
+                // this is a hack
```

Comment:
This is a temporary hack workaround for this https://github.com/migueldeicaza/gui.cs/pull/1437 

Once that PR is live we can get rid of it with a direct assignment